TrackingTools/GsfTracking/python/CkfElectronCandidateMaker_cff.py

```diff
@@ -50,9 +50,6 @@
     updator = 'KFUpdator'
 )
 
-from Configuration.ProcessModifiers.seedingDeepCore_cff import seedingDeepCore
-seedingDeepCore.toModify(TrajectoryBuilderForElectrons, maxPtForLooperReconstruction = cms.double(0.0) )
```

Comment:
Is this removed because the default value for CkfTrajectoryBuilder.maxPtForLooperReconstruction is already 0.0? Is it foreseen that default tracking and deepcore may end up using different thresholds, so it might be better to keep this?

Reply:
I think it must be removed b/c `TrajectoryBuilderForElectrons` is a [`CkfTrajectoryBuilder`](https://github.com/cms-sw/cmssw/blob/caf9b460791f2b01dd6fbc2d8f4d969eda249029/RecoTracker/CkfPattern/python/CkfTrajectoryBuilder_cfi.py#L7), and that does not use a parameter `maxPtForLooperReconstruction` (only [`GroupedCkfTrajectoryBuilder` does](https://github.com/cms-sw/cmssw/blob/caf9b460791f2b01dd6fbc2d8f4d969eda249029/RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc#L167)). This also implies that these two lines were not having any effect up to now (the parameter wasn't actually being used).

Reply:
To be clearer (maybe I was not): this one update is the one that fixes the unit test (b/c `TrajectoryBuilderForElectrons.maxPtForLooperReconstruction` is invalid); it is not related to the "drop type-specs" updates.